controller/stats_reporter.go

```diff
@@ -123,7 +123,7 @@ func (r *reporter) ReportReconcile(duration time.Duration, key, success string)
 	}
 
 	stats.Record(ctx, reconcileCountStat.M(1))
-	stats.Record(ctx, reconcileLatencyStat.M(int64(duration)))
+	stats.Record(ctx, reconcileLatencyStat.M(int64(duration/time.Millisecond)))
```

Comment:
Is this change relative?

Reply:
We were accidentally reporting nanoseconds (duration is in terms of nanoseconds). We wanted to report milliseconds instead. This change fixes that behavior.